colossalai/checkpoint_io/index_file.py

```diff
@@ -148,3 +148,15 @@ def get_checkpoint_file(self, param_name: str) -> str:
         """
         ckpt_path = self.weight_map[param_name]
         return ckpt_path
+    
+    def get_checkpoint_metadata(self) -> str: 
+        """
+        Get the checkpoint index metadata.
+        """
+        return self.metadata
```

Comment:
this method is kind of redundant, since we can just do like this

```python
index_file = CheckpointIndexFile.from_file('/path/to/json')
index_file.metadata

```